vxfreeswitch/voice.py

```diff
@@ -293,6 +355,13 @@ def handle_outbound_message(self, message):
 
         client_addr = message['to_addr']
         client = self._clients.get(client_addr)
+
+        if (client is None and message.get('session_event') ==
+                TransportUserMessage.SESSION_NEW):
+            client = yield self.create_voice_client()
+            yield client.make_call(message['to_addr'])
+            self.register_client(client)
```

Comment:
We can't register the client until it has an address to register it with. This means that we either need to dial out in `connectionMade`, or set the address in `__init__` and have `make_call` use that address. Which one would be better?